rust-toolchain.toml

```diff
@@ -1,4 +1,3 @@
 [toolchain]
-channel = "nightly-2023-10-18"
-components = ["rust-src"]
-
+channel = "nightly"
```

Comment:
Changing to an unspecified nightly version is not ideal. Different developers may end up using different versions of Rust, either because they already had nightly installed, or they installed/updated it at different dates. Some versions may cause unforeseen issues (as has happened in the past).

Instead, I'd strongly suggest selecting a dated nightly version that conforms to the minimum supported rust version of the project.